test/Ocelot.UnitTests/Configuration/UpstreamTemplatePatternCreatorTests.cs

```diff
@@ -215,7 +216,7 @@ public void should_create_template_pattern_that_matches_query_string()
 
         this.Given(x => x.GivenTheFollowingFileRoute(fileRoute))
             .When(x => x.WhenICreateTheTemplatePattern())
-            .Then(x => x.ThenTheFollowingIsReturned($"^(?i)/api/subscriptions/[^/]+/updates(|\\?)unitId={MatchEverything}$"))
+            .Then(x => x.ThenTheFollowingIsReturned($"^(?i)/api/subscriptions/[^/]+/updates(\\/|\\?|$)unitId={MatchEverything}$"))

```

Comment:
I apologize! Why was the unit test's expected value adjusted? It appears that the old template replaced is the same.

Reply:
Since I changed the template pattern in `UpstreamTemplatePatternCreator.cs`  I also had to change these tests